.github/actions/setup-python-poetry/action.yml

```diff
@@ -21,9 +21,18 @@ runs:
       if: runner.os != 'Windows'
       run: |
         set -eux
+        set -o pipefail
         export POETRY_HOME="$HOME/.poetry"
         mkdir -p "$POETRY_HOME"
-        curl -sSL https://install.python-poetry.org | python3 - --version=${{ inputs.poetry-version }}
+        if ! curl -sSL https://install.python-poetry.org | python3 - --version=${{ inputs.poetry-version }}; then
+          logfiles=(${{ github.workspace }}/poetry-installer.log)
```

Comment:
I don't understand why we do a loop while it seems we only dump `poetry-installer.log` :thinking: 

Can't we just do  `cat ${{ github.workspace }}/poetry-installer.log` instead ?

Reply:
:open_mouth: I miss the glob part

Reply:
all good now :)

Reply:
still too complicated for it own good, I would have done `cat ${{ github.workspace }}/poetry-installer*.log` to avoid the tedious loop and array manipulation in shell
(or if you want to display the file name in the dump, you can use this glorious hack `tail -n +1 ${{ github.workspace }}/poetry-installer*.log`

Reply:
Your suggestion is great, I'll use it :D